base/Dockerfile.ubi8

```diff
@@ -109,6 +109,17 @@ RUN microdnf --nodocs install yum \
 ARG SKIP_SECURITY_UPDATE_CHECK="false"
 RUN yum check-update || "${SKIP_SECURITY_UPDATE_CHECK}"
 
+# Install Python 3.6.15 to address CVE-2021-3737
```

Comment:
If you _do_ wind up doing this, I recommend following a different pattern:

* FROM <ubi-image>:<tag> as scratch
* Install all of your build/link time dependencies.
* Download the python TGZ
* Untar it
* `./configure ...`
* `make && make && make install DESTDIR=/some/predefined/install/root`

Then later on you do another

* FROM <ubi-image>:<tag>
* COPY --from=scratch /some/predefined/install/root /usr/local

... then do the rest

This means that the TGZ and any intermediate build files are discarded.